rfcs/0093-rfc-categories.md

```diff
@@ -0,0 +1,152 @@
+---
+feature: rfc_categories
+start-date: 2021-05-19
+author: David Arnold
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @kevnicox, @Mic92, @gytis-ivaskevicius
+shepherd-leader: @kevincox
+related-issues:
+category: process
+---
+
+# Summary
+[summary]: #summary
+
+The RFC process is amended with means to categorize RFCs into one of: _feature_, 
+_information_ or _process_, where each category sets different accents that
+improve the overall outcome of the process.
+
+# Motivation
+[motivation]: #motivation
+
+## Some issues are not addressed by the community
+
+There is no appropriate venue for it. When reading the current RFC process, it becomes
+clear that it was made primarily with purely technical decisions in mind. It might only
+be a perception, but this perception systemically skews what's being discussed and what
+not.
+
+As a result, important topics might not be adressed and settled by the community at large
+in a generally accepted procedure.
```

Comment:
```suggestion
via a generally accepted procedure.
```

Reply:
I might take a different approach and say:

> As a result it is difficult to raise many important topics for discussion since the RFC process often leads reviewers to view the topic in the wrong light.